src/protocol/vm/engine.rs

```diff
@@ -34,6 +38,26 @@ where
     <D as DatabaseRef>::Error: Debug,
 {
     let engine = SimulationEngine::new(db.clone(), trace);
+
+    let zero_account_info =
```

Comment:
@tamaralipows I actually like this!! this way we don't need to re init all of these every time we make a new pool state!